src/Dynatrace.MetricUtils/DynatraceMetricsSerializer.cs

```diff
@@ -181,8 +181,7 @@ private void WriteTimestamp(StringBuilder sb, DateTime timestamp)
 				return;
 			}
 
-
-			sb.Append($" {new DateTimeOffset(timestamp.ToLocalTime()).ToUnixTimeMilliseconds()}");
+			sb.Append($" {timestamp.ToUniversalTime().ToUnixTimeMilliseconds()}");
```

Comment:
IIUC https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.tounixtimemilliseconds?view=net-6.0#system-datetimeoffset-tounixtimemilliseconds correctly the call to `ToUniversalTime()` is redundant here.